### PR TITLE
Nga 451 471 484 485 

### PR DESCRIPTION
**Pull request by jdextercsc**
This Pull Request is for part of NGA-451, namely 471, 484 and 485

Added iptables rules for compute nodes
Added iptables rules for swift proxy nodes
added iptables rules for swift storage rules
added iptables rules for scaleio to all nodes
added reject all rule to all nodes
added template file that provides loopback, ssh and established states for iptables and starts iptables.

```
PLAY RECAP ********************************************************************
auto-deploy-node           : ok=3    changed=0    unreachable=0    failed=0
compute-1                  : ok=103  changed=68   unreachable=0    failed=0
controller-1               : ok=411  changed=299  unreachable=0    failed=0
controller-2               : ok=415  changed=304  unreachable=0    failed=0
controller-3               : ok=415  changed=304  unreachable=0    failed=0
scaleio-1                  : ok=76   changed=50   unreachable=0    failed=0
scaleio-2                  : ok=76   changed=50   unreachable=0    failed=0
scaleio-3                  : ok=74   changed=48   unreachable=0    failed=0
swift-1                    : ok=98   changed=63   unreachable=0    failed=0
swift-2                    : ok=98   changed=63   unreachable=0    failed=0
swift-3                    : ok=98   changed=63   unreachable=0    failed=0
```

### CODE REVIEW

**Comment by jcpowermac** [comment]
Code Review :+1:

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -67,19 +67,29 @@
...
-# setup base ssh and reject rules
-- name: setup ssh iptables rule
-  iptables:
-    port: 22
-    protocol: tcp 
-    comment: "ssh incoming"
+# setup base ssh rules
+- name: turn off firewalld
+  service:
+    name: firewalld
```
Comment (mtnbikenc) [inline roles/common/tasks/main.yml:73]: firewalld is already in openstack-common.  Should it be removed from openstack-common?  And this should probably include `enabled: false`.

File: roles/common/tasks/main.yml [hunk 2]
```diff
@@ -67,19 +67,29 @@
...
+    state: stopped
   tags: iptables
 
-# have to use command as there is no opt field in module yet.
-#- name: setup reject all
-#  command: iptables -A INPUT -j REJECT --reject-with icmp-host-prohibited
-#  tags: iptables
+- name: iptables start
```
Comment (mtnbikenc) [inline roles/common/tasks/main.yml:78]: Could this task be removed and put `enabled: true` in the task after copying to base rules?  Seems redundant to start-copy-restart unless there is something specific required by starting iptables first.

File: roles/common/tasks/main.yml (hunk 1 above)
Comment (jdextercsc) [inline roles/common/tasks/main.yml:73]: Yes we she should remove it from openstack common.

File: roles/common/tasks/main.yml (hunk 2 above)
Comment (jdextercsc) [inline roles/common/tasks/main.yml:78]: I tried it that way, iptables kept failing to load on the first time around.  It does seem redundent, but it I would rather it work each time then be a maybe

File: roles/common/tasks/main.yml (hunk 1 above)
Comment (jdextercsc) [inline roles/common/tasks/main.yml:73]: Do you want the removal from openstack-common to be in this PR or a separate one?

Comment (mtnbikenc) [inline roles/common/tasks/main.yml:73]: Adding a commit to this PR for removing firewalld from openstack-common is fine.

File: roles/common/tasks/main.yml (hunk 2 above)
Comment (mtnbikenc) [inline roles/common/tasks/main.yml:78]: Seems odd, but I'd rather it work too. :)  I'm fine with leaving it as is.

**Comment by mtnbikenc** [comment]
Code review :+1:

**Pull request merged**